<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsNewMemoryLayerDialogBase</class>
 <widget class="QDialog" name="QgsNewMemoryLayerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>293</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>New Temporary Scratch Layer</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Layer name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mNameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="buttonGroup1">
     <property name="title">
      <string>Type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QRadioButton" name="mPointRadioButton">
        <property name="text">
         <string>Point</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="mLineRadioButton">
        <property name="text">
         <string>Line</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="mPolygonRadioButton">
        <property name="text">
         <string>Polygon</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="mMultiPointRadioButton">
        <property name="text">
         <string>Multipoint</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="mMultiLineRadioButton">
        <property name="text">
         <string>Multiline</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="mMultiPolygonRadioButton">
        <property name="text">
         <string>Multipolygon</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QgsProjectionSelectionWidget" name="mCrsSelector" native="true">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&lt;i&gt;&lt;b&gt;Warning:&lt;/b&gt; Temporary scratch layers are not saved and will be discarded when QGIS is closed.&lt;/i&gt;</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QgsProjectionSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">qgsprojectionselectionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mNameLineEdit</tabstop>
  <tabstop>mPointRadioButton</tabstop>
  <tabstop>mLineRadioButton</tabstop>
  <tabstop>mPolygonRadioButton</tabstop>
  <tabstop>mMultiPointRadioButton</tabstop>
  <tabstop>mMultiLineRadioButton</tabstop>
  <tabstop>mMultiPolygonRadioButton</tabstop>
  <tabstop>mCrsSelector</tabstop>
  <tabstop>mButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsNewMemoryLayerDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>387</x>
     <y>304</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsNewMemoryLayerDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
